internal/elasticsearch/index/ilm.go

```diff
@@ -389,6 +389,26 @@ var supportedActions = map[string]*schema.Schema{
 			},
 		},
 	},
+	"downsample": {
+		Description: "Roll up documents within a fixed interval to a single summary document. Reduces the index footprint by storing time series data at reduced granularity.",
+		Type:        schema.TypeList,
+		Optional:    true,
+		MaxItems:    2,
```

Comment:
Why 2? IIUC there's no case where a user should specify more than a single `downsample` block right?